app/routes/api.js

```diff
@@ -13,9 +13,8 @@ const router = new Router();
 
 router.get('/', (req, res) => {
     res.json({
-        message: 'Welcome to the Astro API',
+        message: 'Welcome to the Astro API.',
```

Comment:
Since the API is changing and currently doesn't mirror the original Bitcannon shall we just start from scratch?